README.md

```diff
@@ -45,12 +45,27 @@ Garden Linux is a [Debian](https://debian.org) derivate that aims to provide a s
 
 ## Build Requirements
 
-The entire build runs in a docker container (well a privileged one with extended capabilities - since we need loop back support)
-We can run on any system supporting Docker and having loopback support and has
+The entire build runs in a docker container (well a privileged one with extended capabilities - since we need loop back support).
+We can run on any system supporting Docker and having loopback support and has:
 
-- 2+ GiB (use RAM-disk; use fs with sparse-file support)
+- 2+ GiB (use '--lessram' to lower memory usage)
 - 10+ GiB free disk space
-- Internet connection to access snapshot.debian.org and repo.gardenlinux.io
+- Internet connection (snapshot.debian.org, deb.debian.org, repo.gardenlinux.io, docker.io, golang.org, gopkg.in, github.com)
+
+## Build Options
```

Comment:
Can we move the `Build Options` chapter right before the `Quick Start` chapter? I think this makes more sense since it is now placed directly within the Build Requirements chapter, splitting it into two parts (see `Required packages for a convenient build` for example).

Reply:
Makes sense. Done